File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        for(int i=0; i<myString.length(); i++) {
            if(myString.indexOf(pat,i)== -1){
                break;
            }else{
                i = myString.indexOf(pat,i);
                answer++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181871. 문자열이 몇 번 등장하는지 세기/문자열이 몇 번 등장하는지 세기.cs
using System;

public class Solution {
    public int solution(string myString, string pat) {
        int answer = 0;
        for(int i=0;i<myString.Length;i++){
            if(myString.IndexOf(pat,i) == -1){
                break;
            }else{
                i = myString.IndexOf(pat,i);
                answer++;
            }
        }
        return answer;
    }
}